kiwi/bootloader/config/grub2.py

```diff
@@ -271,12 +270,14 @@ def setup_disk_boot_images(self, boot_uuid, lookup_path=None):
             by the distribution. The bios core image is created
             when grub2-install is called
         """
-        log.info('Creating grub bootloader images')
+        self.__copy_theme_data_to_boot_directory(lookup_path)
 
-        if self.firmware.efi_mode():
-            self.efi_boot_path = self.create_efi_path()
+        if not self.firmware.efi_mode():
+            return
 
-        self.__copy_theme_data_to_boot_directory(lookup_path)
+        log.info('Creating grub bootloader images')
+
+        self.efi_boot_path = self.create_efi_path()
 
```

Comment:
we need a font but not modules, fonts are cross platform basically, modules are efi specific